packages/discovery-provider/integration_tests/tasks/test_index_user_bank.py

```diff
@@ -216,6 +227,78 @@ def test_process_user_bank_tx_details_valid_purchase(app):
         assert buyer_transaction_record.tx_metadata == str(track_owner_id)
 
 
+def test_process_user_bank_tx_details_valid_purchase_album(app):
```

Comment:
Oof I wish we had just migrated existing balance purchases to go through payment router so you didn't have to update both indexers.